metals/src/main/scala/scala/meta/internal/builds/GradleBuildTool.scala

```diff
@@ -60,7 +60,7 @@ case class GradleBuildTool() extends BuildTool {
        |              scalaCompilerPlugin
        |          }
        |          dependencies {
-       |              scalaCompilerPlugin 'org.scalameta:semanticdb-scalac_' + scalaLib.version + ':4.1.4'
+       |              scalaCompilerPlugin 'org.scalameta:semanticdb-scalac_' + scalaLib.version + ':4.1.12'
```

Comment:
Ups, we should probably do `${BuildInfo.semanticdbVersion}` - I should have done it from the start. Can you change it here?

Reply:
Sure, I was about to yesterday, but I wasn't sure. Thanks for confirming.